src/routes/cart.routes.ts

```diff
@@ -15,6 +22,12 @@ cartRouter.get("/", [protectRoute], viewCart);
 
 cartRouter.delete("/remove/:id", [protectRoute], removeFromCart);
 
-cartRouter.delete("/clear", [protectRoute], clearCart);
+cartRouter.delete("/clear", [isAuth], clearCart);
+
+cartRouter.patch(
+  "/update/:id",
+  [protectRoute, ValidateJoi(ProductSchema.product.updateCart)],
```

Comment:
```suggestion
  [isAutth, ValidateJoi(ProductSchema.product.updateCart)],
```

Reply:
No need bro, because this middleware called "protectRoute" have the same functionality as "isAuth" + "twoFactorAuth" ( when is enabled )